doc/manual/command-ref/conf-file.xml

```diff
@@ -468,6 +468,14 @@ builtins.fetchurl {
     this option to <literal>true</literal>.</para></listitem>
   </varlistentry>
 
+  <varlistentry xml:id="conf-low-speed-limit"><term><literal>low-speed-limit</literal></term>
```

Comment:
Maybe change this to something like `stalled-download-timeout`? `low-speed-limit` is pretty vague (it's not even clear what speed it refers to).